source/_includes/custom/after_footer.html

```diff
@@ -1,3 +1,53 @@
 {% comment %}
-  Add content to be output at the bottom of each page. (You might use this for analytics scripts, for example)
 {% endcomment %}
+
+<script type="text/javascript">
+  // made header sticky
+  var sticky = new Waypoint.Sticky({
+    element: $("#banner"),
+    offset: -100
+  })
+
+  {% if page.opensource == true %}
+    // render open source projects
+    var groupedRepositories = _.groupBy(repositories, function(repository){ return repository.category })
+    _.each(groupedRepositories, function(categorizedRepositories, index){
+      var collection = new Backbone.Collection(categorizedRepositories);
+      new RepositoriesView({collection: collection, el: $("#" + collection.models[0].attributes.category)});
```

Comment:
I wonder if you could actually just render this all "server-side" now that we punted on the Github auth. That would just involve using ruby to iterate through the repositories.json instead. Wouldn't hold up the PR for it :)